src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -479,6 +479,7 @@ std::string ContextAttribute::render
       }
 
       out += valueTag(indent + "  ", ((ciP->outFormat == XML)? "contextValue" : "value"),
+                        "value",
```

Comment:
Yes, this set of functions 'valueTag' are a nightmare.
Overloading functions used to be my favorite C++ feature, among with default values for parameters.
Right now I feel both these features are really bad ideas.

I will take a look at this particular change.
However, if the functests aren't complaining ...  This is a VERY centric piece of code - rendering of an attribute ...